### ISSUE TITLE
End Game and click balance

### ISSUE THREAD

**Issue by torrentx** [particle-clicker/particle-clicker#95]
Things I've noticed once research and employees are raised to around level 50.

-Click value of data is in the 100M while each second 100B or so data is produced, making clicking a waste of time.  
-There are no more upgrades after this period
-Achieving the Level 100 in research or employees, takes days of just leaving the browser open with no user interaction.  

Proposed fixes:
-Have achievements end at Level 50 instead of 100
-Add in additional upgrades to make achieving level 100 more user interactive (make clicks worth a lot more than second research)

Realistically, I think ending the game for level 50 (capping this too) in achievements. Is the way to go, the game feel pretty balanced up to that point.

**Comment by Tampa** [comment]
Adding additional workers with reputation to them or achievements yielding reputation and thus funding(you know the cycle) should balance out the endgame.

**Comment by Quix0r** [comment]
Yes, I am at that level now (just after 14 hours of playing/sleeping, means leaving browser tab open).

**Comment by kdungs** [comment]
IIRC we just threw the achievements on top without giving it too much thought re balancing. Feel free to send a PR.
